react/components/WordpressPage.tsx

```diff
@@ -0,0 +1,171 @@
+/* eslint-disable @typescript-eslint/camelcase */
+import React, { FunctionComponent } from 'react'
+import { Helmet } from 'react-helmet'
+import { useQuery } from 'react-apollo'
+import { useRuntime } from 'vtex.render-runtime'
+import { Spinner } from 'vtex.styleguide'
+import { Container } from 'vtex.store-components'
+import insane from 'insane'
+import { useCssHandles } from 'vtex.css-handles'
+
+import SinglePageBySlug from '../graphql/SinglePageBySlug.graphql'
+import Settings from '../graphql/Settings.graphql'
+
+const sanitizerConfig = {
+  allowedTags: [
+    'h1',
+    'h2',
+    'h3',
+    'h4',
+    'h5',
+    'h6',
+    'blockquote',
+    'p',
+    'a',
+    'ul',
+    'ol',
+    'nl',
+    'li',
+    'b',
+    'i',
+    'strong',
+    'section',
+    'em',
+    'strike',
+    'code',
+    'hr',
+    'br',
+    'div',
+    'table',
+    'thead',
+    'caption',
+    'tbody',
+    'tr',
+    'th',
+    'td',
+    'pre',
+    'img',
+    'iframe',
+    'figure',
+  ],
+  allowedAttributes: {
+    a: ['href', 'name', 'target'],
+    img: ['src', 'alt'],
+    iframe: ['src', 'scrolling', 'frameborder', 'width', 'height', 'id'],
+  },
+  allowedSchemes: ['http', 'https', 'mailto', 'tel'],
+}
+
+const sanitizerConfigStripAll = {
+  allowedAttributes: false,
+  allowedTags: false,
+  allowedSchemes: [],
+}
+
+const CSS_HANDLES = [
+  'postFlex',
+  'postContainer',
+  'postTitle',
+  'postMeta',
+  'postFeaturedImage',
+  'postBody',
+  'postChildrenContainer',
+] as const
+
+const WordpressPage: FunctionComponent = _props => {
+  const handles = useCssHandles(CSS_HANDLES)
+  const {
+    route: { params },
+  } = useRuntime()
+  const { loading: loadingS, data: dataS } = useQuery(Settings)
+  const { loading, error, data } = useQuery(SinglePageBySlug, {
+    variables: { slug: params.slug },
+  })
+
+  if (loading || loadingS) {
+    return (
+      <div className="mv5 flex justify-center" style={{ minHeight: 800 }}>
+        <Spinner />
+      </div>
+    )
+  } else if (error) {
+    return (
+      <div className="ph5" style={{ minHeight: 800 }}>
+        Error! {error.message}
+      </div>
+    )
+  } else if (data?.wpPages?.pages) {
```

Comment:
You can simplify these conditions and let the happy path be the last return making it easier to read.

```suggestion
  }
  if (error) {
    return (
      <div className="ph5" style={{ minHeight: 800 }}>
        Error! {error.message}
      </div>
    )
  }
  if (!data?.wpPages?.pages) {
    return (
      <div>
        <h2>No page found.</h2>
      </div>
    )
  }
```

Reply:
I just added a new commit that I think addresses this by re-ordering the if/else statements